cl8-web/server/scripts/clearUsersandDatabase.js

```diff
@@ -66,26 +57,26 @@ async function main() {
 
     if (result.confirmation !== domainToCheck) {
       console.log("This response doesn't match. Exiting early")
-      process.exit("exiting early a they don't match")
+      process.exit("exiting early as they don't match")
     }
 
     console.log('clearing accounts and data:')
     // - remove all the user accounts
 
     try {
-      await clearFirebaseAccounts()
-    } catch (e) {
-      console.log
+      await clearFirebaseAccounts();
+    } catch (err) {
+      console.log(err);
```

Comment:
Is `debug(err)` better? Same thing on line 76.

Reply:
Stuff with `debug` will only appear if you set the corresponding DEBUG env var. When this is failing, I figured it's more helpful to see why this is failing.

My thinking being that if you're running this on the CLI, you are more prepared to read a stacktrace, and you might not think to have set `DEBUG="cl8.*"` first.